.github/workflows/tests.yml

```diff
@@ -62,43 +70,21 @@ jobs:
           from os.path import basename
           from re import compile
           main = '^main$'
-          release = '^v\d+\.\d+\.\d+$'
+          release = 'release_\d+\_\d+\_\d+$'
```

Comment:
We want release branches to be named release_0_0_9 so that they don't conflict with tags (ReadTheDocs gets confused).

Reply:
If you add a `^` at the beginning of the pattern, it'll make sure the branch starts with `release`.

Reply:
Added